src/main/java/bo/CartActions.java

```diff
@@ -1,38 +1,40 @@
 package bo;
 
+import com.google.inject.Inject;
 import elements.PageElement;
 import factory.DriverFactory;
-import org.testng.Assert;
 import page.CartWebPage;
 
 import java.util.List;
 import java.util.stream.Collectors;
 
 public class CartActions {
-    private final CartWebPage boxPage;
+    @Inject
+    private final CartWebPage cartPage;
 
-    public CartActions() {
-        boxPage = new CartWebPage();
+    public CartActions(CartWebPage cartWebPage) {
```

Comment:
Why do you need this Constructor

Reply:
becouse I should initisialize cartPage

Reply:
you dount need constructor at all. Pages should be initializated without constructors

Reply:
fixed